cinema/views.py

```diff
@@ -0,0 +1,42 @@
+from django.shortcuts import render, get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.templatetags.rest_framework import data
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    if request.method == "POST":
+        serializer = MovieSerializer(data, request.data)
+        if serializer.is_valid():

```

Comment:
```suggestion
        serializer.is_valid(raise_exception=True)
```

This will handle 400 error automatically